app-config.prod.yaml

```diff
@@ -28,3 +28,19 @@ auth:
       production:
         clientId: ${AUTH_GITHUB_CLIENT_ID}
         clientSecret: ${AUTH_GITHUB_CLIENT_SECRET}
+
+kubernetes:
+  serviceLocatorMethod:
+    type: 'multiTenant'
+  clusterLocatorMethods:
+    - type: 'config'
+      clusters:
+        - url: https://api.smaug.na.operate-first.cloud:6443
+          name: smaug
+          authProvider: 'serviceAccount'
+          skipTLSVerify: false
+          skipMetricsLookup: false
+          # The plugin will automatically use the auto mounted token
+          # serviceAccountToken: ${SERVICE_ACCOUNT_TOKEN}
```

Comment:
I'm not sure about this (but I haven't tested it). Do you know for sure that backstage knows where to find the token?

Reply:
I think it gets it from `/var/run/secrets/kubernetes.io/serviceaccount/token`, but I'll test it out again just to make sure and I'll try to find a reference in the code just be sure.